lang/stata/stata.py

```diff
@@ -0,0 +1,170 @@
+from talon import Context, actions, settings
+
+ctx = Context()
+
+ctx.matches = r"""
+code.language: stata
+"""
+
+# functions.py
+ctx.lists["user.code_parameter_name"] = {
+    # regressions
+    "v c e cluster": "vce(cluster)",
+    "v c e robust": "vce(robust)",
+}
+
+# functions_common.py
+ctx.lists["user.code_common_function"] = {
+    # base stata
+    "global": "global",
+    "local": "local",
+    "reg": "reg",
+    "regress": "reg",
+    # packages
+    "estadd": "estadd",
+    "estout": "estout",
+    "estpost": "estpost",
+    "eststo": "eststo",
+    "esttab": "esttab",
+}
+
+# libraries_gui.py
+ctx.lists["user.code_libraries"] = {
+    "estout": "estout",
+}
+
+
+@ctx.action_class("user")
+class UserActions:
+    # comment_line.py
+    def code_comment_line_prefix():
+        actions.auto_insert("* ")
+        # actions.auto_insert("// ")
```

Comment:
Can delete this comment?

Reply:
I removed this line. There are multiple ways to comment out a line in stata

Reply:
Thanks. I don't think there is a good way atm to supoort both, you'd probably have to use a setting. But probably easier to just have a linter automatically change them. 

fwiw if one of them is for multi-line comments (like C style /* ... */) there is a separate comment action called code_comment_block you could use, but it looks like it's probably not the case here.

Reply:
Stata uses the default c-style comment! That should already be activated.

Most stata users use `*`  for comments at the start of a line, `//`  for comments at the end of a line and `/*...*/`  for long multiline comments.  Note that  can also be used at the start of a line whereas `*`  cannot be used at the end of a line.